### PR TITLE
Update ko/tutorial/keras

### PR DESCRIPTION
**Pull request by rickiepark**
Updates for recent changes in en folder.
Running with Colab is Ok.
Thanks! :)

### CODE REVIEW

File: site/ko/tutorials/keras/basic_classification.ipynb [hunk 1]
```diff
@@ -49,6 +49,8 @@
     "#\n",
     "# Copyright (c) 2017 François Chollet\n",
     "#\n",
+    "# IGNORE_COPYRIGHT: cleared by OSS licensing\n",
```
Comment (lamberta) [inline site/ko/tutorials/keras/basic_classification.ipynb]: I think you can ignore this for now. We don't like the way this looks and are working with the OSS team to make this better

File: site/ko/tutorials/keras/basic_regression.ipynb [hunk 2]
```diff
@@ -49,6 +49,8 @@
     "#\n",
     "# Copyright (c) 2017 François Chollet\n",
     "#\n",
+    "# IGNORE_COPYRIGHT: cleared by OSS licensing\n",
```
Comment (lamberta) [inline site/ko/tutorials/keras/basic_regression.ipynb]: Same. Remove from this PR, please

File: site/ko/tutorials/keras/basic_text_classification.ipynb [hunk 3]
```diff
@@ -49,6 +49,8 @@
         "#\n",
         "# Copyright (c) 2017 François Chollet\n",
         "#\n",
+        "# IGNORE_COPYRIGHT: cleared by OSS licensing\n",
```
Comment (lamberta) [inline site/ko/tutorials/keras/basic_text_classification.ipynb]: Remove from this PR, please

File: site/ko/tutorials/keras/overfit_and_underfit.ipynb [hunk 4]
```diff
@@ -59,6 +59,8 @@
     "#\n",
     "# Copyright (c) 2017 François Chollet\n",
     "#\n",
+    "# IGNORE_COPYRIGHT: cleared by OSS licensing\n",
```
Comment (lamberta) [inline site/ko/tutorials/keras/overfit_and_underfit.ipynb]: Remove from this PR, please

File: site/ko/tutorials/keras/save_and_restore_models.ipynb [hunk 5]
```diff
@@ -49,6 +49,8 @@
     "#\n",
     "# Copyright (c) 2017 François Chollet\n",
     "#\n",
+    "# IGNORE_COPYRIGHT: cleared by OSS licensing\n",
```
Comment (lamberta) [inline site/ko/tutorials/keras/save_and_restore_models.ipynb]: Remove from this PR, please

**Comment by rickiepark** [comment]
Thanks for your comment, I remove the tags.

**Comment by lamberta** [comment]
Hi Haesun,
You'll need to sync basic_regression for the changes to `plot_history` from https://github.com/tensorflow/docs/pull/305

Also, all notebook translations now require the `unicode_literals` module, just include it with the other imports like:

```
from __future__ import absolute_import, division, print_function, unicode_literals
```

THanks

**Comment by rickiepark** [comment]
Hi Billy,
I added unicode_literals to notebooks in ko/tutorial/keras and updated plot_history function in basic_regression.ipynb.
Running with Colab is Ok.

I'm trying to translate notebooks in en/r2/tutorial/keras.
What do you think of it?

Thanks.

**Comment by lamberta** [comment]
Thanks, Haesun!

As for r2/, yes, the r2/tutorials/keras notebooks should be stable since we're just migrating the r1 notebooks over.

But we'll need to make sure the latest changes to the r1/ notebooks get cherrypicked into r2/. I haven't been good at making sure people update both places :/

**Comment by rickiepark** [comment]
Thanks @lamberta 
I'm trying to diff en/tutorial/keras to en/r2/tutorial/keras.
If necessary, I will make a PR to en/r2/tutorial/keras.

**Pull request merged**